ecosystem/python/sdk/aptos_sdk/account_address.py

```diff
@@ -167,3 +210,97 @@ def test_token(self):
             base_address, "bob's collection", "bob's token"
         )
         self.assertEqual(actual, expected)
+
+    def test_to_standard_string(self):
+        # Test special address: 0x0
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000000"
+            ).to_standard_string(),
+            "0x0",
+        )
+
+        # Test special address: 0x1
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000001"
+            ).to_standard_string(),
+            "0x1",
+        )
+
+        # Test special address: 0x4
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000004"
+            ).to_standard_string(),
+            "0x4",
+        )
+
+        # Test special address: 0xf
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x000000000000000000000000000000000000000000000000000000000000000f"
+            ).to_standard_string(),
+            "0xf",
+        )
+
+        # Test special address from short no 0x: d
+        self.assertEqual(
+            AccountAddress.from_hex("d").to_standard_string(),
+            "0xd",
+        )
+
+        # Test non-special address from long:
+        # 0x0000000000000000000000000000000000000000000000000000000000000010
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000010"
+            ).to_standard_string(),
+            "0x0000000000000000000000000000000000000000000000000000000000000010",
```

Comment:
nit: For all of these testing, I would assign the addr string to a var. Kinda hard to eyeball them, especially the one with 0s

Reply:
Fair call!